src/Fixer/FunctionNotation/PhpdocToReturnTypeFixer.php

```diff
@@ -32,7 +32,7 @@
 final class PhpdocToReturnTypeFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
     /**
-     * @var array<array<int, string>>
+     * @var array<array<int|string>>
```

Comment:
Looks wrong. IMO before the change it was correct

Reply:
No,
`array<int|string>` means an array of `int` and `string` elements,
`array<int, string>` means an array indexed by `int` with values of `string`, but in the 2nd level array we have `T_STRING` which is `int`.

Maybe we could go with `@var array<int, array<int|string>>`.

Reply:
If I remember correctly, `int` is assumed when no array index type is provided with thix syntax, so I would keep the current annotation.

Reply:
the `array-key` type is assumed, which also allows strings, see https://phpstan.org/r/1d784a07-1eb6-49c8-ac4e-a76995d877c5

Reply:
I'll add `int` key type then, thanks for checking.